content/release-10-14-0/announcements-10-14-0.md

```diff
@@ -23,13 +23,13 @@ As of release 10.16+, new tenants can utilize a new enhanced time-series support
 The following APIs will be removed without any replacement:
 
 * `GET /measurement/measurements/{id}`
-* `DEL /measurements/measurement/getById`
+* `DEL /measurements/measurement/{id}`
 
-The following API will no longer be supported:
+The following API is not supported in 10.16 will be supported starting with 10.17:
```

Comment:
```suggestion
The following API is not supported in release 10.16 but will be supported starting with release 10.17:
```

Reply:
I would need to adapt again unfortunately